playbook/app/pb_kits/playbook/pb_message/docs/_message_hover.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= pb_rails("message", props: {
+  avatar_name: "Mike Bishop",
+  avatar_status: "online",
+  avatar_url: "https://randomuser.me/api/portraits/men/50.jpg",
+  border_radius: "rounded",
+  hover: { background: "success_subtle" },
+  label: "Anna Black",
+  message: "How can we assist you today?",
+  padding: "xs",
+}) %>
+
+<br><br>
+
+<%= pb_rails("message", props: {
+  align_timestamp: "left",
+  avatar_name: "Lucille Sanchez",
+  avatar_url: "https://randomuser.me/api/portraits/women/50.jpg",
+  border_radius: "rounded",
+  hover: { shadow: "deepest" },
+  label: "Becca Jacobs",
+  message: "Application for Kate Smith is waiting for your approval",
+  padding: "xs",
+}) %>
+
+<br><br>
```

Comment:
Can we remove the br since we do not have more content after this line?

Reply:
Was matching the react example, so I'll leave it for now. 

And we can update both at a later point if need be. I wanted to make sure we had some spacing in there a little, to give the examples a little breathing room.

Reply:
We should use our global spacing props instead of br tags for spacing in our docs